src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -843,7 +843,11 @@ void NonClientIslandWindow::_UpdateFrameMargins() const noexcept
         HPAINTBUFFER buf = BeginBufferedPaint(hdc.get(), &rcRest, BPBF_TOPDOWNDIB, &params, &opaqueDc);
         if (!buf || !opaqueDc)
         {
-            winrt::throw_last_error();
+            // MSFT:34673647 - BeginBufferedPaint can fail, but it probably
+            // shouldn't bring the whole Terminal down with it. So don't
+            // throw_last_error here.
+            LOG_LAST_ERROR();

```

Comment:
I wonder -- will this become garrulous? If we fail to paint, and then try again, again, again, again...

Reply:
We're still going to `return 0` out the bottom, so theoretically we'll still be telling the OS that we did handle the paint.

The real fix to that bug though might be figuring out more about however simself is running Terminal tests, cause whatever they're doing seems to cause these miscellaneous crashes.